File: Reception/Models/Session.cs
using System;
using System.Collections.Generic;

namespace Reception.Models;

public partial class Session
{
    public int Id { get; set; }

    public int UserId { get; set; }

    public string Code { get; set; } = null!;

    public DateTime CreatedAt { get; set; }

    public DateTime ExpiresAt { get; set; }

    public virtual Account User { get; set; } = null!;
}

File: Garden/src/app/core/types/session.ts
import { Client } from './client';
import { Account } from './account';

export type Session = {
    id: number,
    accountId: number,
    clientId: number,
    /**
     * Max Length: 36
     * Min Length: 0
     */
    code: string | null,
    createdAt: Date,
    expiresAt: Date,
    account: Account,
    client: Client,
}